controller/articles.controller.js

```diff
@@ -16,6 +16,16 @@ exports.getArticlesById = (req, res, next) => {
     .then((articles) => {
       res.status(200).send(articles);
     })
+    .catch(next);
+};
 
+exports.patchArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  const newVote = req.body;
+
+  patchArticle(article_id, newVote)
+    .then((article) => {
+      res.status(200).send(article);
```

Comment:
It's best practise to send back our response in object on an appropriate key 🔑 
You could do this (it'll mean also needing to update your tests to look on this key) but it's not mandatory